src/config.ts

```diff
@@ -54,3 +54,46 @@ export async function resolveConnections(): Promise<ConnectionConfig[]> {
 
   return result;
 }
+
+/**
+ * Validate connection keys.
+ *
+ * @param {string[]} keys
+ * @returns {void}
+ */
+function validateConnections(keys: string[]): void {
+  const errors: string[] = [];
+  keys.forEach(element => {
+    if (!process.env[element]) {
+      errors.push(element);
+    }
+  });
+  if (errors.length) {
+    throw new Error(`Following environment variables were not set: ${errors}`);
+  }
+}
+
+/**
+ * Resolve database connections from Env.
+ *
+ * @param {string[]} keys
+ * @returns {ConnectionConfig[]}
+ */
+export function resolveConnectionsFromEnv(): ConnectionConfig[] {
+
+  validateConnections(ENV_KEYS);
+
+  const connection = {
+    client: process.env.DB_CLIENT,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    user: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME,
+    options: {
+      encrypt: process.env.DB_ENCRYPTION || false
+    }
+  };
+
+  return [connection as ConnectionConfig];
```

Comment:
Make the above change and you can make it simple.
```suggestion
  return [connection];
```